lessons/12-css-pseudo-selectors/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+
+<header>
+  <title>Pseudo Classes</title>
+</header>
```

Comment:
This should be `<head>`, instead of `<header>`

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header

Reply:
Done